### PR TITLE
Add support for the new sharing configuration

### LINKED ISSUE #133: Update sharing flow for PSPDFKit 8 for iOS
https://github.com/PSPDFKit/react-native/pull/131/commits/6ae5e0616260b8747e0336cea5738754a4b99607 removed support for the old sharing flow, which was changed in PSPDFKit 8 for iOS.

This wrapper should be updated to work with the new sharing APIs: https://pspdfkit.com/guides/ios/current/miscellaneous/document-sharing/

### PR DESCRIPTION
**Pull request by OscarSwanros**
Fixes #133 

---

This PR adds support for the new sharing flow on PSPDFKit 8 for iOS.

### CODE REVIEW

**Comment by radazzouz** [comment]
Merged #131 so I am changing the destination branch to master.

File: samples/Catalog/ios/Catalog.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -532,8 +532,8 @@
 			isa = PBXGroup;
 			children = (
 				01107DE51FFE52D800CD9D66 /* PSPDFKitUI.framework */,
-				657278751D86C28800A5E1A8 /* RCTPSPDFKit.xcodeproj */,
 				6572778F1D83184800A5E1A8 /* PSPDFKit.framework */,
+				657278751D86C28800A5E1A8 /* RCTPSPDFKit.xcodeproj */,
```
Comment (radazzouz) [inline samples/Catalog/ios/Catalog.xcodeproj/project.pbxproj]: Strange... Why did this happen?

File: samples/Catalog/Catalog.ios.js [hunk 2]
```diff
@@ -78,6 +78,26 @@ var examples = [
       });
     }
   },
+  {
```
Comment (radazzouz) [inline samples/Catalog/Catalog.ios.js]: We should use prettier here to properly indent the Javascript code.

Comment (radazzouz) [reply samples/Catalog/Catalog.ios.js]: I'd also move this example at the very bottom.

File: samples/Catalog/ios/Catalog.xcodeproj/project.pbxproj (hunk 1 above)
Comment (OscarSwanros) [reply samples/Catalog/ios/Catalog.xcodeproj/project.pbxproj]: Reordered in the inspector.

**Pull request merged**